plugins/MantisGraph/core/graph_api.php

```diff
@@ -184,11 +196,17 @@ function graph_pie( array $p_metrics, $p_title = '' ) {
  * Cumulative line graph
  *
  * @param array   $p_metrics      Graph Data.
+ * @param integer $p_wfactor      Width factor for graph chart. Eg: 2 to make it double wide
  * @return void
  */
-function graph_cumulative_bydate( array $p_metrics ) {
+function graph_cumulative_bydate( array $p_metrics, $p_wfactor = null ) {
```

Comment:
`$p_wfactor` = 1 instead of the # set default code

Reply:
Ok, i can change it. I applied the same reason as the previous comment.
Will address it in some later commit with more small fixes